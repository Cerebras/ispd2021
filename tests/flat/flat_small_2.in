2
4 4
2
4 4
# this is a comment
1 0
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1
1