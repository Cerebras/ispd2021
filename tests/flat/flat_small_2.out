1
0 0 0 1 2
0 2 0 1 2
compute_map:
0 0
0 1
0 2
0 3
adapter_map:
